examples/EncodeDecode/main.roc

```diff
@@ -0,0 +1,111 @@
+app "example"
+    packages {
+        pf: "https://github.com/roc-lang/basic-cli/releases/download/0.8.1/x8URkvfyi9I0QhmVG98roKBUs_AZRkLFwFJVJ3942YA.tar.br",
+        json: "https://github.com/lukewilliamboswell/roc-json/releases/download/0.6.3/_2Dh4Eju2v_tFtZeMq8aZ9qw2outG04NbkmKpFhXS_4.tar.br",
+    }
+    imports [pf.Stdout.{ line }, json.Core.{ json }]
+    provides [main] to pf
+
+### start snippet impl
+
+ItemKind := [
+    Text,
+    Method,
+    Function,
+    Constructor,
+    Field,
+    Variable,
+    Class,
+    Interface,
+    Module,
```

Comment:
Would be really cool to make this have 10 values to force at least one to decode 2 digits